notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11755,6 +11755,73 @@ fear [There were pointed moment when Jillian was in terror of her husband, Spenc
 incorporeal being [Spencer's body had been taken over by a malevolent alien entity that, at least according to Sherman Reese, was traveling space as a transmission at the speed of light.]
 
 
+movie: Bicentennial Man (1999)
+==============================
+
+:: Title
+Bicentennial Man
+
+:: Description
+A robot housekeeper grows more and more human-like over the course of his 200
+year life. It is based on the 1992 novel The Positronic Man by Isaac Asimov
+and Robert Silverberg (which is itself based on Asimov's original 1976
+novelette "The Bicentennial Man").
+
+:: Date
+1999-12-17
+
+:: References
+https://en.wikipedia.org/wiki/Bicentennial_Man_(film)
+
+:: Ratings
+2 <paul>
+
+:: Genre
+sci-fi
+
+:: Choice Themes
+the desire to become more human-like [Central to the story is the humanoid robot Andrew and his quest to become a human being. He went from being a naive, metallic, humanoid robot to acquiring the ability to eat, feel emotions and sensations, and even have sexual relationships, resulting in him and Portia falling in love.]
+
+:: Major Themes
+mechanical man [The NDR series robot Andrew is the main character in the story.]
+master and slave [Much was made of the Martin family robot housekeeper Andrew having no rights and being the property of the family. He was later granted his freedom, and ultimately declared to be a human being by the World Congress.]
+friendship [Andrew maintained a lifelong friendship with Amanda, whom he'd help raise from the time she was a little girl.]
+android [Andrew became a bonafide android after receiving an upgrade midway through the film.]
+AI prejudice [never be accepted]
```

Comment:
Incomplete entry?

Reply:
Whoops. Fixed. Actually, I'm mildly surprised that's the only one I muffed given the sheer number of cliched themes featured.